### PR TITLE
Use CDN to load azure blob browser lib

### PR DESCRIPTION
**Pull request by bion**
We've been using https://webjars.org to load javascript libraries (just one so far) but the azure blob storage library specifically causes a Java compilation error on the latest patch version of Java 11.

This change is to unblock upgrading to Java 11.0.14 [while we figure out how to resolve the webjar problem](https://github.com/webjars/webjars/issues/1967).

Unblocks https://github.com/seattle-uat/civiform/pull/2316, https://github.com/seattle-uat/civiform/issues/2315

### CODE REVIEW

File: universal-application-tool-0.0.1/app/views/AzureFileUploadViewStrategy.java [hunk 1]
```diff
@@ -105,7 +105,7 @@ public Tag renderFileUploadBlockSubmitForms(
     return div(formTag, skipForms, buttons)
         .withId("azure-upload-form-component")
         .with(
-            footer(viewUtils.makeWebJarsTag(WebJarJsPaths.AZURE_STORAGE_BLOB)),
```
Comment (clouser2) [inline universal-application-tool-0.0.1/app/views/AzureFileUploadViewStrategy.java:108]: Looks good. Is it worth creating a separate issue tracking reverting to this path once the Azure issue is fixed upstream? We could then add a TODO against it here.

**Review by clouser2 — APPROVED** [review]
Nice!

File: universal-application-tool-0.0.1/app/views/AzureFileUploadViewStrategy.java (hunk 1 above)
Comment (bion) [reply universal-application-tool-0.0.1/app/views/AzureFileUploadViewStrategy.java:108]: Done.

**Comment by bion** [comment]
oof, the browser test that's failing in CI is passing when i run it on my local machine. will look into it tomorrow.

**Review by clouser2 — APPROVED** [review]

**Pull request merged**